src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/Grammar/GrammarGenerator.cs

```diff
@@ -74,17 +74,23 @@ public static string Run(List<TreeType> types)
                 }
             }
 
+            // Add some rules not present in Syntax.xml.
+            rules.Add("Utf8StringLiteralToken", [Join(" ", [RuleReference("StringLiteralToken"), RuleReference("Utf8Suffix")])]);
+            rules.Add("Utf8MultiLineRawStringLiteralToken", [Join(" ", [RuleReference("MultiLineRawStringLiteralToken"), RuleReference("Utf8Suffix")])]);
+            rules.Add("Utf8SingleLineRawStringLiteralToken", [Join(" ", [RuleReference("SingleLineRawStringLiteralToken"), RuleReference("Utf8Suffix")])]);
+            rules.Add("Utf8Suffix", [new("'u8'"), new("'U8'")]);
```

Comment:
adds a few pseudo rules to make it so that hte generated grammar has less 'see lexical specificatoin' productions.